part1.js

```diff
@@ -537,4 +538,83 @@ document.addEventListener("DOMContentLoaded", () => {
   }
   document.addEventListener("keydown", show);
 
+  appleIn.ondragstart = function() {
```

Comment:
Turn off the default drag for the image. Not doing this will most likely cause the drag to stick and you cannot let go.